homeassistant/components/n26/sensor.py

```diff
@@ -0,0 +1,288 @@
+        sensor_entities.append(N26Space(api, space))
+
+    async_add_entities(sensor_entities)
+
+
+class N26Account(Entity):
+    """Sensor for a N26 balance account.
+
+    A balance account contains an amount of money (=balance). The amount may
+    also be negative.
+    """
+
+    def __init__(self, client, account_info,
+                 account_statuses, limits, balance) -> None:
+        """Initialize a N26 balance account."""
+        self._client = client
+        self._account_info = account_info
+        self._account_statuses = account_statuses
+        self._limits = limits
+        self._balance = balance
+
+    @property
+    def should_poll(self) -> bool:
+        """Return True.
+
+        Data needs to be polled from the bank servers.
+        """
+        return True
+
+    def update(self) -> None:
+        """Get the current balance and currency for the account."""
+        self._account_info = self._client.get_account_info()
+        self._account_statuses = self._client.get_account_statuses()
+        self._limits = self._client.get_account_limits()
+        self._balance = self._client.get_balance()
+
+    @property
+    def unique_id(self):
+        """Return the unique ID of the entity."""
+        return self._balance["iban"]
+
+    @property
+    def name(self) -> str:
+        """Friendly name of the sensor."""
+        return "n26_%s" % self._balance["iban"].lower()
+
+    @property
+    def state(self) -> float:
+        """Return the balance of the account as state."""
+        return self._balance["bankBalance"]
+
+    @property
+    def unit_of_measurement(self) -> str:
+        """Use the currency as unit of measurement."""
+        return self._balance["currency"]
+
+    @property
+    def device_state_attributes(self) -> dict:
+        """Additional attributes of the sensor."""
+        attributes = {
+            ATTR_IBAN: self._balance.get("iban"),
+            ATTR_AVAILABLE_BALANCE: self._balance.get("availableBalance"),
+            ATTR_USABLE_BALANCE: self._balance.get("usableBalance"),
+            ATTR_ACC_OWNER_TITLE: self._account_info.get("title"),
+            ATTR_ACC_OWNER_FIRST_NAME: self._account_info.get("kycFirstName"),
+            ATTR_ACC_OWNER_LAST_NAME: self._account_info.get("kycLastName"),
+            ATTR_ACC_OWNER_GENDER: self._account_info.get("gender"),
+            ATTR_ACC_OWNER_BIRTH_DATE: timestamp_ms_to_date(
+                self._account_info.get("birthDate")),
+            ATTR_ACC_OWNER_EMAIL: self._account_info.get("email"),
+            ATTR_ACC_OWNER_PHONE_NUMBER:
+                self._account_info.get("mobilePhoneNumber"),
+        }
+
+        for limit in self._limits:
+            limit_attr_name = ("limit_%s" % limit["limit"]).lower()
+            attributes[limit_attr_name] = limit["amount"]
+
+        return attributes
+
+    @property
+    def icon(self) -> str:
+        """Set the icon for the sensor."""
+        return ICON_ACCOUNT
```

Comment:
So sad we cannot change icon according the currency unit 💵

Just joking... no need to implement it.